.github/workflows/go.yml

```diff
@@ -70,9 +70,10 @@ jobs:
     name: lint
     runs-on: ubuntu-latest
     steps:
+      - uses: actions/checkout@v4
       - uses: actions/setup-go@v5
         with:
           go-version: 1.18
-      - uses: actions/checkout@v4
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v4
+        uses: golangci/golangci-lint-action@v6
```

Comment:
no harm in bumping this dependency, but lint job fails. Created #1411 to address fixing the lint job